third-party/realdds/include/realdds/dds-device-server.h

```diff
@@ -81,6 +81,8 @@ class dds_device_server
     void on_set_option( set_option_callback callback ) { _set_option_callback = std::move( callback ); }
     void on_query_option( query_option_callback callback ) { _query_option_callback = std::move( callback ); }
 
+    std::unordered_map< std::string, std::shared_ptr< dds_stream_server > > & get_streams() { return _stream_name_to_server; }
```

Comment:
It's just called `streams()` in my version

Reply:
(and I made it a `map`

Reply:
I will rename `streams()`.

`unordered_map` gives you O(1) access, why is `map` better here?